ejb-txn-remote-call/server/extensions/postgresql-module.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module name="org.postgresql.jdbc" xmlns="urn:jboss:module:1.8">
     <resources>
-        <artifact name="org.postgresql:postgresql:42.2.8"/>
+        <artifact name="org.postgresql:postgresql:42.2.27"/>
```

Comment:
Updated the version because the old version will lead to Exception with Hibernate.